cvmfs/swissknife_sync.h

```diff
@@ -292,6 +295,11 @@ class CommandSync : public Command {
     r.push_back(Parameter::Optional('S',
                                     "virtual directory options "
                                     "[snapshots, remove]"));
+    r.push_back(
+        Parameter::Optional('G', "Use persistent caching for all catalogs "
```

Comment:
For switches, we already have infrastructure code (see next parameter).
```suggestion
        Parameter::Switch('G', "Use persistent caching for all catalogs "
```